README.md

```diff
@@ -320,6 +322,61 @@ i18n.offChangeLocale(fn);
 i18n.onceChangeLocale(fn);
 ```
 
+## Pluralization
+
+With this package, you can translate with pluralization. To do that you need to use the pipe separator `|` in your locale translations, like in the example below:
+
+```yml
+_locale: 'en'
+phone: 'zero phones | one phone | two phones | {$count} phones'
+```
+
+Then just add the `count` argument to the `getTranslation` options:
+
+```js
+i18n.getTranslation('phone', { count: 0 }); // -> zero phones
+i18n.getTranslation('phone', { count: 1 }); // -> one phone
+i18n.getTranslation('phone', { count: 2 }); // -> two phones
+i18n.getTranslation('phone', { count: 3 }); // -> 3 phones
+i18n.getTranslation('phone', { count: 1000 }); // -> 1000 phones
+```
```

Comment:
_(I'll comment here because otherwise it'd be in a random helper.)_

I have two problems with the current approach:
1. **It's backward-incompatible.** I checked and we use `{$count}` 6 times in one of the projects, and it'll work as long as nobody will put ` | ` in there.
2. **It's mixing interpolation with pluralization.** While I agree that `{$count}` is better than `{count}`, it won't work with custom interpolation (i.e., with custom `open` and `close`).

I see two ways of fixing both at the same time, but there are definitely more:
1. Add a new `getPluralizedTranslation` function, which will effectively call `getTranslation` (which solves custom interpolation) and only then apply the pluralization.
2. Call it `_count` instead of `count` to stay in align with `_locale` and `_namespace`. It's still backward-incompatible, but probably won't affect anyone. Plus, the pluralization logic should be applied _after_ interpolation too.

I think I like 2. more, but it's up to you, of course.

Additionally, I think the pluralization should be a separate function, just like `_interpolateTranslation`, so one can override it with custom logic. Of course, the ` | ` separator should be configurable as well.